src/control/cmd/dmg/utils.go

```diff
@@ -146,3 +147,28 @@ func formatHostGroupResults(buf *bytes.Buffer, groups hostlist.HostGroups) strin
 
 	return buf.String()
 }
+
+// tabulateHostGroups is a helper function representing hostgroups in a tabular form.
+func tabulateHostGroups(groupTitle string, columnTitles []string, groups hostlist.HostGroups) (string, error) {
+	titles := []string{groupTitle}
+	titles = append(titles, columnTitles...)
+
```

Comment:
This could be simpler if you make the titles param variadic:

```go
func tabulateHostGroups(groups hostlist.HostGroups, titles ...string) (string, error) {
        if len(titles) < 2 {
                return "", errors.New("incorrect number of titles")
        }
...
```